sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/ft_personalize_3.c

```diff
@@ -37,6 +65,9 @@ static status_t personalize(ujson_t *uj) {
   LOG_INFO("Waiting for FT provisioning data ...");
   TRY(ujson_deserialize_manuf_cert_perso_data_in_t(uj, &in_data));
 
+  // Configure certificat flash info page permissions.
```

Comment:
s/certificat/certificate/. In a follow up PR if this is green and ready to submit.